superset/viz.py

```diff
@@ -1376,6 +1376,7 @@ def query_obj(self):
         return d
 
     def get_data(self, df):
+        self.form_data['metrics'] = [self.form_data.get('metric')]
```

Comment:
Ok makes sense to me now, let's just change line 1399 from ` values=self.get_metric_label(fd.get('metric')))` to  values=self.get_metric_label([fd.get('metric')]))